src/app/pages/subjects/subjects.js

```diff
@@ -122,7 +122,7 @@ function Subjects({model}) {
             filter: newCategory,
             path: pagePath
         });
-        document.title = (categoryEntry && 'title' in categoryEntry) ? categoryEntry.title : model.title;
+        document.title = (categoryEntry && categoryEntry.title) || model.title;
     }
```

Comment:
FYI, tutor's started using [optional chaining](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining) https://github.com/openstax/tutor-js/blob/main/babel.config.js#L16  which would allow `categoryEntry?.title || model.title`